source/reference/limitations.txt

```diff
@@ -0,0 +1,73 @@
+.. _c2c-limitations:
+
+===========
+Limitations
+===========
+
+.. include:: /includes/preview-warning
+
+{+c2c-full-product-name+} is under active development. This preview
+release has several limitations.
+
+General Limitations
+-------------------
+
+- The minimum supported version is MongoDB 6.0.
+- The minimum support Feature Compatibility Version is 6.0. 
+- MongoDB Community Edition is not supported.
+- The source and destination clusters must have the same release
+  version.
+- Environmental assumptions are not validated as pre-checks. 
+- The Primary cluster cannot be changed while syncing.
+- If write blocking is disabled,the client must prevent writes to the
+  source cluster after starting the commit process.
+- DDL operations are not supported during syncing. 
+- Filtered synchronization is not supported.
+- Network compression is not supported.
+- applyOps operations from the source cluster are not supported.
+
+Unsupported Collection Types
+----------------------------
+
+- Collections with collation are not supported.
```

Comment:
collation and capped will be supported at GA, but not time series

Reply:
As above, we will update this doc for GA to remove limits that are no longer valid

Reply:
hmm, I worry that this will give the impression to some people that their use case isn't supported and they won't look back in a month to see that their use case is now supported.

Reply:
see comment above

Reply:
Updated with "currently ..." as suggested above